packages/microsoft_exchange_server/data_stream/httpproxy/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,334 @@
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.handlercompletionlatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.handlertomoduleswitchinglatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.httppipelinelatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.httpstatus"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.kerberosauthheaderlatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.majorversion"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.minorversion"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.moduletohandlerswitchinglatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.requestbytes"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.requesthandlerlatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.responsebytes"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.revisionversion"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.routinglatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.totalaccountforestlatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.totalglslatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.totalrequesttime"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.totalresourceforestlatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.totalsharedcachelatency"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.activitycontextlifetime"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.httpproxyoverhead"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- convert: 
+    field: "microsoft.exchange.proxytime"
+    type: long
+    ignore_failure: true
+    ignore_missing: true
+- set:
+    field: event.ingested
+    value: "{{{_ingest.timestamp}}}"
+    ignore_failure: true
```

Would be nice to add on failure processors for each pipeline to be able to debug any errors easily

```suggestion
    ignore_failure: true
on_failure:
  - set:
      field: event.kind
      value: pipeline_error
  - append:
      field: error.message
      value: '{{{ _ingest.on_failure_message }}}'
```